en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/A/Alias_Magnificent_Mansion.md

```diff
@@ -26,7 +26,7 @@ _7th-level conjuration_
 **Components:** V, S, M (a miniature portal carved from ivory, a small piece of polished marble, and a tiny silver spoon, each item worth at least 5 gp) \
 **Duration:** 24 hours
 
-You conjure an extradimensional residence within range.
```

Comment:
A5e removed "within range" for many spells. If we want to be consistent, the fact that it states a range in the top part of the spell _should_ be sufficient. So I removed it from all A and B spells, unless there was a good reason not to.